elastic/kibana
# [Infra] Add Flyout to table view in Infrastructure Inventory

## Summary

Closes #201138 

This PR adds a Flyout when clicking an item in the table view for the Infrastructure Inventory, as we had it in the Map view but the table view was missing.

### How to test
1. Start Kibana and fill it with data, `node scripts/synthtrace infra_hosts_with_apm_hosts` can be used if you don't have data.
2. Navigate to Infrastructure Inventory
3. Change the view to table view
4. Click on an item, the flyout should appear

### Screenshots
Before|After
-|-
![image](https://github.com/user-attachments/assets/585a4036-364e-447c-9f27-130e594b7a69)|![image](https://github.com/user-attachments/assets/4fc42ee7-27ed-4035-b24d-6bcc305407e5)
